### ISSUE TITLE
Microsoft Learn - progress continuing course

### ISSUE THREAD

**Issue by JustinK1973** [MicrosoftDocs/feedback#721]
Hi, Microsoft learn is a great resource, however when part way through a module and leave it is very difficult to work out where you were especially if doing more than 1 at same time.

Could we under our profile or main page have listing of not completed courses so we can continue.

**Comment by sstjean** [comment]
I'd like to second this recommendation.  It's really hard to figure out where you stopped.

**Comment by hansmbakker** [comment]
I added a thumbs up 👍

**Comment by MVDB1984** [comment]
It would indeed be nice to have a progress like on MVA

**Comment by Regenhardt** [comment]
Yeah much is missing from MVA. Learning was honestly simpler there, and that is not about being used to something since I've used learn for some time now, but it doesn't get better.
